### DISCUSSION TITLE
How to adjust spacing before table of contents and chapter headings?

### DISCUSSION THREAD

**Discussion by bjarkeaf** [joaomlourenco/novathesis#334] [Q&A]
First of all, thanks for a cool template! 

My question:
Is it possible to start table of contents and chapter headings at the very top of the page? 
And is this possible while maintaining a spacing for pages in the front matter like abstract and acknowledgements?

Here is an illustration of what I am trying to achieve:
![image](https://user-images.githubusercontent.com/39918665/170702728-feb58f2a-15c1-4d17-898a-0bea522b1ced.png)

**Comment by joaomlourenco** [comment]
Which chapter style are you using?
Do you want to apply the same procedure to the numbered chapters?

**Reply by bjarkeaf** [reply]
I believe it is the default "bar". Yes, I would also like to apply it to chapters.

**Reply by joaomlourenco** [reply]
The select another chapter style.  :)
Try "compact" and see if it fits your needs.  If it does not, try the other ones.  Then let me know which one is closer to your needs and what would you like different.  Maybe I can help…

**Comment by joaomlourenco** [comment]
Add the following to a file named `NOVAthesisFiles/ChapterStyle/bar-compact.ldf`
```latex
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% ChapterStyle/bar-compact.ldf
%% NOVA thesis configuration file
%%
%% Customization for the “NOVA School of Science and Technology” chapter style
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\typeout{NT FILE ChapterStyle/bar-compact.ldf}%

\input{NOVAthesisFiles/ChapStyles/bar.ldf}
\setlength{\beforechapskip}{-20pt}
```
and then open `Config/1_novathesis.tex` and (near line 116) and replace the line
```latex
%\ntsetup{chapstyle=elegant}   % The chapter style to be used
```
with
```latex
\ntsetup{chapstyle=bar-compact}   % The chapter style to be used
```

**Reply by SquirlZz** [reply]
Hi! I intend to do so as well but only for the Annexes, how can i do it without affecting the rest of the document? Thank you in advance!
